src/content/app/genome-browser/components/browser-cog/BrowserCogList.tsx

```diff
@@ -29,27 +29,40 @@ import BrowserCog from './BrowserCog';
 
 import {
   getBrowserCogList,
-  getBrowserCogTrackList,
   getBrowserSelectedCog
 } from 'src/content/app/genome-browser/state/track-config/trackConfigSelectors';
 import {
   CogList,
-  updateCogTrackList,
+  updateCogList,
   updateSelectedCog
 } from 'src/content/app/genome-browser/state/track-config/trackConfigSlice';
+import {
+  getBrowserActiveFocusObjectId,
+  getBrowserActiveGenomeId
+} from '../../state/browser-general/browserGeneralSelectors';
```

Comment:
Change the import path to `src/content/app/genome-browser/state/browser-general/browserGeneralSelectors`?